### PR TITLE
[#169591795] Add specifications for missing error responses

### PR DESCRIPTION
**Pull request by alexgpeppe**
This PR aims to add the missing error responses to the OpenAPI specifications file.

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🐞 [#169591795](https://www.pivotaltracker.com/story/show/169591795): Nel file di specifiche OpenAPI alcune risposte di errore non sono documentate.

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b743345ff911290aa6291d6cd88be9f2e2151b96
</p>

File: openapi.yml [hunk 1]
```diff
@@ -220,6 +235,8 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/UserProfile"
+        404:
```
Comment (ioggstream) [inline openapi.yml]: afaik status codes should be string, see https://swagger.io/docs/specification/describing-responses/

```suggestion
        "404":
```

File: openapi.yml [hunk 2]
```diff
@@ -244,6 +261,16 @@ paths:
...
                 $ref: '#/components/schemas/AdministrationSearchResult'
 components:
+  responses:
+    403Forbidden:
+      description: Forbidden
+      <<: *problem+json-content
+    409Conflict:
+      description: Conflict
+      <<: *problem+json-content
+    500ErrorInternal:
```
Comment (ioggstream) [inline openapi.yml]:
```suggestion
    500InternalServerError:
```

if we use http message, then always do it.

File: openapi.yml [hunk 3]
```diff
@@ -182,9 +197,9 @@ paths:
               schema:
                 $ref: '#/components/schemas/BinaryFile'
         403:
-          description: Forbidden
+          $ref: "#/components/responses/403Forbidden"
         404:
-          description: File not found
+          $ref: https://teamdigitale.github.io/openapi/0.0.5/definitions.yaml#/responses/404NotFound
```
Comment (ioggstream) [inline openapi.yml]: I suggest moving all `teamdigitale.github.io` refs to `components/responses` and reference responses from there.

**Pull request merged**